### ISSUE TITLE
Cache usage are not listing all items

### ISSUE THREAD

**Issue by onlinebizsoft** [Smile-SA/magento2-module-debug-toolbar#26]
When we log all cache call from different cache classes, we can collect around 300 cache calls on the page but the tool only show around 40 calls. There should be a document for what this cache usage tab shows to avoid confusion.

**Comment by guvra** [comment]
Hi,

I just checked, the cache information is properly displayed.

In the "last x executions" tab, are you sure that you selected the execution related to the main page?
The execution selected by default is the latest execution, and it's often an ajax request, not the main page itself.

For example:

![debug_toolbar_executions](https://github.com/Smile-SA/magento2-module-debug-toolbar/assets/3967259/47d22106-fabd-4ac9-8079-0aaa50f35edb)

To see the stats of the main page, I have to select the 4th execution (catalog_category_view).

**Comment by onlinebizsoft** [comment]
@guvra yes I'm very clear with that and I view the correct execution correctly
![image](https://github.com/Smile-SA/magento2-module-debug-toolbar/assets/22501570/4d1397f4-2f6c-4c23-b0bb-0079da81ea47)
![image](https://github.com/Smile-SA/magento2-module-debug-toolbar/assets/22501570/cfdb4342-5ef1-4ee1-a81c-f29a600058cd)

A lots of cache items are missing for this page for example layout, system config,.......you may double check this cache key which any Magento installation should have APP_ACTION_LIST and it is missing in the cache identifier list

**Comment by guvra** [comment]
I did some debugging.

Some cache identifiers are not displayed because the plugin of our module is based on `Magento\Framework\App\CacheInterface` instead of `Magento\Framework\Cache\FrontendInterface`.

Unfortunately, it's not possible to create a plugin on `FrontendInterface`.
Adding a plugin on this interface would trigger the following error:

```
Fatal error: Uncaught LogicException: Circular dependency: Magento\Framework\Interception\PluginList\PluginList depends on Magento\Framework\Interception\PluginList\PluginList and vice versa. in vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php on line 55
```

This is because the plugin system has a dependency on FrontendInterface.
So adding a plugin on this interface creates an infinite loop.

Additionally, I don't think there's a lot of value to display a list of system cache ids (translations, configs...).
The toolbar already shows if these cache types are enabled or disabled.
